### PR TITLE
Bug 1043953 - Set `origin` field in manifest.webapp files. +autoland

### PR DESCRIPTION
**Pull request by ochameau**
https://bugzilla.mozilla.org/show_bug.cgi?id=1043953

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a2cc756ca551)

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9b8ed6f25d89)

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=efa8d5db10c5)

File: build/post-manifest.js [hunk 1]
```diff
@@ -2,60 +2,20 @@
...
-  }
-  var manifestContent = utils.getJSON(stageManifest);
-  if (manifestContent.connections) {
-    manifestContent =
-      manifestInterAppHostnames(manifestContent, options);
-    utils.writeContent(stageManifest, JSON.stringify(manifestContent));
-  }
+  // Forces the presence of `origin` field in order to help WebIDE overriding
+  // the app, with the same origin.
+  manifest.origin = webapp.url;
```
Comment (RickyChien) [inline build/post-manifest.js]: I cannot figure out that why we need to overwrite origin again? (I saw your patch already overwrite it in webapp-manifests.js)

**Pull request merged**